sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_visits_daily_v2/query.sql

```diff
@@ -0,0 +1,302 @@
+    metrics.boolean.pocket_enabled AS organic_content_enabled,
+    metrics.boolean.pocket_sponsored_stories_enabled AS sponsored_content_enabled,
+    metrics.boolean.topsites_sponsored_enabled AS sponsored_topsites_enabled,
+    metrics.boolean.topsites_enabled AS organic_topsites_enabled,
+    metrics.boolean.newtab_search_enabled AS newtab_search_enabled,
+    category AS event_category,
+    name AS event_name,
+    extra AS event_details,
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop_stable.newtab_v1`,
+    UNNEST(events)
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+    -- selecting the events we need to compute on
+    -- Do we need to include any other events for computing "all_visits"?
+    AND (
+      category = 'newtab'
+      AND name IN ('opened')
+      OR category = 'newtab.search'
+      AND name IN ('issued')
+      OR category = 'newtab.search.ad'
+      AND name IN ('click')
+      OR category = 'pocket'
+      AND name IN (
+        'click',
+        'impression',
+        'dismiss',
+        'thumb_voting_interaction',
+        'thumb_voting_interaction'
+      )
+      OR category = 'topsites'
+      AND name IN ('click', 'impression', 'dismiss')
+      OR category = 'newtab'
+      AND name IN (
+        'weather_change_display',
+        'weather_open_url_provider',
+        'weather_location_selected'
+      )
+      OR category = 'newtab'
+      AND name IN (
+        'wallpaper_click',
+        'wallpaper_category_click',
+        'wallpaper_highlight_cta_click',
+        'wallpaper_highlight_dismissed'
+      )
+      OR category = 'newtab'
+      AND name IN (
+        'topic_selection_open',
+        'topic_selection_dismiss',
+        'topic_selection_topics_saved'
+      )
+      OR category = 'newtab'
+      AND name IN (
+        'sections_block_section',
+        'sections_follow_section',
+        'sections_unblock_section',
+        'sections_unfollow_section'
+      )
+      OR category = 'newtab'
+      AND name IN ('inline_selection_click')
+    )
+)
+SELECT
+  submission_date,
+  client_id,
+  mozfun.map.get_key(event_details, 'newtab_visit_id') AS newtab_visit_id,
+  -- ANY_VALUE: because we do not expect those fields to differ within a visit
+  ANY_VALUE(app_version) AS app_version,
+  ANY_VALUE(os) AS os,
+  ANY_VALUE(channel) AS channel,
+  ANY_VALUE(locale) AS locale,
+  ANY_VALUE(browser_version) AS browser_version,
+  ANY_VALUE(country) AS country,
+  ANY_VALUE(newtab_homepage_category) AS newtab_homepage_category,
+  ANY_VALUE(newtab_newtab_category) AS newtab_newtab_category,
+  ANY_VALUE(organic_content_enabled) AS organic_content_enabled,
+  ANY_VALUE(sponsored_content_enabled) AS sponsored_content_enabled,
+  ANY_VALUE(sponsored_topsites_enabled) AS sponsored_topsites_enabled,
+  ANY_VALUE(organic_topsites_enabled) AS organic_topsites_enabled,
+  ANY_VALUE(newtab_search_enabled) AS newtab_search_enabled,
```

Comment:
This is a change from the current `newtab_visits`, but it is **not a problem**.  I ran the numbers to see how often you'd have multiple, distinct values here.  Almost never happens thankfully, country and version are the most common with about .006% of the visits yesterday

Reply:
The [newtab_visits_v1](https://github.com/mozilla/bigquery-etl/blob/9061e61b78fb3db4e8a0f04d7fac393d2de1ba73/sql/moz-fx-data-shared-prod/telemetry_derived/newtab_visits_v1/query.sql#L31-L50) appears to do a similar thing of using `ANY_VALUE`.
The _expectation_ is for those values to not change within the same visit

Reply:
Correct, just validated it myself and wanted to leave a note